webdriver/tests/interface.html

```diff
@@ -1,49 +1,23 @@
 <!doctype html>
 <meta charset=utf-8>
-
+<link rel="help" href="https://w3c.github.io/webdriver/webdriver-spec.html">
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script src=/resources/WebIDLParser.js></script>
 <script src=/resources/idlharness.js></script>
 
-<script type=text/plain class=untested>
-[Exposed=Window]
-interface Navigator {
-  // objects implementing this interface also implement the interfaces given below
-};
-</script>
-
-<script type=text/plain>
-Navigator includes NavigatorAutomationInformation;
-
-interface mixin NavigatorAutomationInformation {
-  readonly attribute boolean webdriver;
-};
-</script>
-
 <script>
-"use strict";
-
-test(() => assert_idl_attribute(navigator, "webdriver"), "navigator.webdriver is present");
```

Comment:
`assert_idl_attribute` is already covered by `idlarray.test()` below - hence, redundant, and deleted.